### DISCUSSION TITLE
Minecraft crashed how to backbup?

### DISCUSSION THREAD

**Discussion by t4nertemel** [itzg/docker-minecraft-server#1726] [Q&A]
Hi all, I'm using Ubuntu 22.04.1 LTS and have installed and Docker and the Minecraft was working well for a few days then the game crashed loosing all data.

My question is how can I backup the world so that if it crashed again I can restore it.

Many thanks
Taner

**Comment by itzg** [comment]
There is a companion backup image 

https://github.com/itzg/docker-mc-backup

**Reply by t4nertemel** [reply]
Thank you for this itzg but to be honest I don't know how to start with installing and configuring this, is there any tutorials you can point me to so that I get started with installing and using docker-mc-backup?

Many thanks

**Reply by itzg** [reply]
I don't know of any, but there's probably some around. 

Otherwise, there's an example file provided there https://github.com/itzg/docker-mc-backup/blob/master/examples/docker-compose.yml
